src/main/java/com/mattermost/integration/figma/api/comment/service/PostCommentServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.mattermost.integration.figma.api.comment.service;
+
+import com.mattermost.integration.figma.api.comment.dto.PostCommentRequestDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class PostCommentServiceImpl implements PostCommentService {
+
+    private static final String BASE_URL = "https://api.figma.com/v1/files/%s/comments";
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Override
+    public void postComment(String fileId, String replyCommentId, String message, String token) {
+        String url = String.format(BASE_URL, fileId);
+
+        var postCommentRequestDTO = new PostCommentRequestDTO();
```

Comment:
Please use the appropriate data type here (PostCommentRequestDTO)

Reply:
fixed